### DISCUSSION TITLE
Upgrade from Pimcore 6.8 to Pimcore X

### DISCUSSION THREAD

**Discussion by AleksandarV993** [pimcore/pimcore#14759] [Q&A / Support] [answered]
So I've read the update guide for Pimcore update V6 to V10 [this one](https://pimcore.com/docs/pimcore/10.2/Development_Documentation/Installation_and_Upgrade/Updating_Pimcore/V6_to_V10.html)
And I've done everything as suggested by the guide. I've first successfully updated from Pimcore v6.8 to v6.9.6. After that I re-did the folder structure as per instructions for Pimcore X projects. The installation (composer update) passed smooth, but I've got the following issue:

```
An exception occurred while executing 'SELECT * FROM settings_store WHERE id = ? AND scope = ?' with params ["BUNDLE_INSTALLED__Wvision\\Bundle\\DataDefinitionsBundle\\DataDefinitionsBundle", "pimcore"]:

SQLSTATE[42S02]: Base table or view not found: 1146 Table 'db-pimcore.settings_store' doesn't exist
```

The table definitely does not exist. After searching through the vendor folder, I've noticed that the migration for that table is located in `install.sql` file which is located in `vendor/pimcore/pimcore/bundles/InstallBundle/Resources/` yet the migration itself was never performed. 
Does anyone know the reason behind this? Has anyone encountered a similar error when updating to higher version of Pimcore? 

I've also tried running `./bin/console doctrine:migrations:migrate --prefix=Pimcore\\Bundle\\InstallBundle` manually, yet I get this error returned: `[ERROR] The version "latest" couldn't be reached, there are no registered migrations.`

**Comment by blankse** [comment, marked as answer]
See the update notes in the 6.9 branch:
https://github.com/pimcore/pimcore/tree/6.9/doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore

The command for the migrations is:
`bin/console pimcore:migrations:migrate -s pimcore_core -n`

The migrations are in the CoreBundle. The migration which create the settings_store db table is here:
https://github.com/pimcore/pimcore/blob/6.9/bundles/CoreBundle/Migrations/Version20210303165115.php

**Reply by AleksandarV993** [reply]
That definitely did the trick, thanks for the help.
I'll mark this as solved.
